jekyll/_config.yml

```diff
@@ -136,4 +136,4 @@ analytics:
     go_button_clicked: go-button-clicked
 rollbar_env: development
 
-server_version: "2.18.3"
+# server_version: "2.18.4"
```

Comment:
removing this for now as it's not working in the PDFs - I have a fix in mind but won't be ready for Server release :-(